airqo-platform/AirQo-frontend
# [WEBSITE] Monitor Page

#### Summary of Changes (What does this PR do?)

- Design of monitor page (1/4 of products pages)
- Navigation through header
- Page assets

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- Click _Products_ in homepage top bar nav to view its dropdown menu.
- Click Binos Monitor to view Monitor page.
- Change your browser window to check for responsiveness of components

#### What are the relevant tickets?

- [WEB-378]

#### Screenshots 
![image](https://user-images.githubusercontent.com/53012890/224056665-f985c8e6-be69-43d9-954c-79be246a3b1f.png)

![image](https://user-images.githubusercontent.com/53012890/224056822-644bb58c-7091-4ce0-86b2-77f559a3715e.png)



[WEB-378]: https://airqoteam.atlassian.net/browse/WEB-378?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
